packages/react-dom/src/__tests__/ReactRenderDocument-test.js

```diff
@@ -54,15 +54,15 @@ describe('rendering React components at document', () => {
       const body = testDocument.body;
 
       ReactDOM.hydrate(<Root hello="world" />, testDocument);
-      expect(testDocument.body.innerHTML).toBe('Hello world');
+      expect(testDocument.body.textContent).toBe('Hello world');
```

Comment:
tests that assert contents of `body` will now find `\<!--%--\>` insertion marker, even left behind after unmounting. I've updated tests to ignore this marker or otherwise account for it (this is one example of that)

Reply:
This is no longer true